db/models.py

```diff
@@ -46,3 +53,76 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        AUTH_USER_MODEL,
+        related_name="orders",
+        on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    row = models.PositiveSmallIntegerField()
+    seat = models.PositiveSmallIntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="row_seat_movie_session"
+            )
+        ]
+
+    def clean(self) -> None:
+        if not (1 <= self.row <= self.movie_session.cinema_hall.rows):
+            raise ValidationError(
+                {
+                    "row": f"row number must be in available range: (1, rows):"
+                           f" (1, {self.movie_session.cinema_hall.rows})"
+                }
+            )
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            raise ValidationError(
+                {
+                    "seat": f"seat number must be in available range: "
+                            f"(1, seats_in_row): (1, "
+                            f"{self.movie_session.cinema_hall.seats_in_row})"
+                }
+            )
+
+    def save(
+        self,
+        force_insert: bool = False,
+        force_update: bool = False,
+        using: str = None,
+        update_fields: str = None
+    ) -> None:
+        self.full_clean()
+        return super(Ticket, self).save(force_insert, force_update)
```

Comment:
add other args to `save`.
Also, there is no need to write `Ticket, self` in `super()`